docs/filewrapper.rst

```diff
@@ -31,24 +31,24 @@ object which supports *at least* the ``read()`` method, and the ``read()``
 method must support an optional size hint argument and the ``read()`` method
 *must* return **bytes** objects (never unicode).  It *should* support the
 ``seek()`` and ``tell()`` methods.  If it does not, normal iteration over the
-filelike object using the provided block_size is used (and copying is done,
-negating any benefit of the file wrapper).  It *should* support a ``close()``
-method.
+``filelike_object`` using the provided ``block_size`` is used (and copying is
```

Comment:
"filelike object" is wrong. It should be either "file-like object" when used in plain old English or "``filelike_object``" when referring to programming language.

In this instance, the latter is seems more contextually appropriate to me, since it aligns with ``block_size``